php/Blocks/Controls/Multiselect.php

```diff
@@ -69,7 +63,7 @@ public function register_settings() {
 				'label'    => __( 'Default Value', 'genesis-custom-blocks' ),
 				'type'     => 'textarea_array',
 				'default'  => '',
-				'help'     => __( 'Enter each default value on a new line.', 'genesis-custom-blocks' ),
+				'help'     => __( 'Each default value on a new line.', 'genesis-custom-blocks' ),
```

Comment:
To be similar to the other text. I removed this word from the other text to display on 2 lines instead of 3.